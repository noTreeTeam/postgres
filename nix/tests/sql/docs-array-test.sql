-- testing sql found in https://supabase.com/docs/guides/database/arrays

create table arraytest (
  id integer not null,
  textarray text array
);

INSERT INTO arraytest (id, textarray) VALUES (1, ARRAY['Harry', 'Larry', 'Moe']);;

select * from arraytest;

SELECT textarray[1], array_length(textarray, 1) FROM arraytest;

drop table arraytest cascade;
