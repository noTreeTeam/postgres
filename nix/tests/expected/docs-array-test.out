-- testing sql found in https://supabase.com/docs/guides/database/arrays
create table arraytest (
  id integer not null,
  textarray text array
);
INSERT INTO arraytest (id, textarray) VALUES (1, ARRAY['Harry', 'Larry', 'Moe']);;
select * from arraytest;
 id |     textarray     
----+-------------------
  1 | {Harry,Larry,Moe}
(1 row)

SELECT textarray[1], array_length(textarray, 1) FROM arraytest;
 textarray | array_length 
-----------+--------------
 Harry     |            3
(1 row)

drop table arraytest cascade;
